Application/registerationForm.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Form Application</title>
+    <link rel="stylesheet" href="registerationForm.css">
+  </head>
+  <body>
+    <div id ="registerationForm">
+      <h1>Student Form </h1>
+      <p>(Note : Asterisk (*) fields should be filled.)</p>
```

Comment:
There is no need to specifically mention this; it is a common practice to use an asterisk symbol as a superscript to indicate that a field is required.